redash/query_runner/cass.py

```diff
@@ -7,8 +7,9 @@
 logger = logging.getLogger(__name__)
 
 try:
-    from cassandra.cluster import Cluster, Error
+    from cassandra.cluster import Cluster
     from cassandra.auth import PlainTextAuthProvider
+    from cassandra import DriverException
```

Comment:
I think that the Cassandra driver version we have in the requirements file doesn't have this :-( Can you confirm?